pkg/test/istio.io/tasks/security/authz-http/wait.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I have a more generic script for this in my PR:
https://github.com/istio/istio/pull/16140/files#diff-e16fe624e2bb22d6623359179084277c
It waits for all containers in all pods (except Jobs) to be ready.

Reply:
Could you put it in a common directory? I can update this PR to use it.

Reply:
I'm not sure which PR will be merged first. ;)